src/security/kuzzleSecurity.js

```diff
@@ -503,82 +456,50 @@ KuzzleSecurity.prototype.profileFactory = function(id, content) {
 /**
  * Get a specific user from kuzzle using its unique ID
  *
- * Takes an optional argument object with the following property:
- *    - hydrate (boolean, default: true):
- *         if is set to false, return a list id in role instead of KuzzleRole.
- *
  * @param {string} id
- * @param {object} [options] - (optional) arguments
  * @param {responseCallback} cb - returns Kuzzle's response
  */
-KuzzleSecurity.prototype.getUser = function (id, options, cb) {
+KuzzleSecurity.prototype.getUser = function (id, cb) {
```

Comment:
The `options` parameter should stay: it can contain a `queued` option, handling offline mode.
See: http://kuzzle.io/sdk-documentation/#getuser